classes/PodsData.php

```diff
@@ -1766,9 +1766,8 @@ public function fetch ( $row = null, $explicit_set = true ) {
             $mode = 'id';
             $id = pods_absint( $row );
 
-            if ( !is_numeric( $row ) || 0 === strpos( $row, '0' ) || $row != preg_replace( '/[^0-9]/', '', $row ) ) {
-                $mode = 'slug';
-                $id = $row;
+            if ( ! is_numeric( $row ) || 0 === strpos( $row, '0' ) || $row != preg_replace( '/[^0-9]/', '', $row ) ) {
```

Comment:
I think we need a `null !== $row check here && ( .... )` otherwise we'll lose the ability to do the slug lookup mode.

Reply:
any reason this is still open? @pcfreak30 @sc0ttkclark

Reply:
never late than never, added a null check and fixed merge block.